containers/datalab/content/setup-env.sh

```diff
@@ -15,12 +15,14 @@
 
 # Sets up various environment variables within the docker container.
 
-export DATALAB_USER=`gcloud -q config list --format yaml | grep account | awk -F" " '{print $2}'`
-export DATALAB_PROJECT_ID=`gcloud -q config list --format yaml | grep project | awk -F" " '{print $2}'`
-if [ -z $DATALAB_PROJECT_NUM ]; then
-  export DATALAB_PROJECT_NUM=`curl --silent -H "Metadata-Flavor=Google" http://metadata.google.internal/computeMetadata/v1beta1/project/numeric-project-id`
+if [ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] && [ -z "$GCLOUDSDK_CONFIG" ]
+then
+  export CLOUDSDK_CONFIG=/content/datalab
```

Comment:
I wonder if we can hard code the path in nodejs to be "/content/datalab" if DATALAB_ENV="local", else "/root/.config/gcloud" for GCE run? It seems the location doesn't need to be customized by users?